### DISCUSSION TITLE
Retraction/Wiping/Z-hop issue

### DISCUSSION THREAD

**Discussion by Rickthebig** [OrcaSlicer/OrcaSlicer#4688] [General]
2.0 Beta
I had to enable Z hop due to printing tall models and I got stringing. I then engaged in investigating different options and found that Z hop is always done "normal" (straight), at least I could not see any spiral mode or anything else, neither while actually printing nor in the preview. Choosing different types of Z hop won't help against stringing at this point.

Trying to use wiping to fight stringing, I also noticed that the percent of "retraction before wipe" will always be 85% if the value is less than 85%. I was hoping to do wiping and retraction simultaneously by setting 0% there, but nope.

I could confirm my findings by reviewing reported bugs, which are still shown as open.  

Of course I have performed the built-in retraction test but found that the test itself runs better than the actual print. The test says 0.7 mm retraction, but I actually need 2.0 mm. 

Then I tried to identify the settings of the calibration model but found nothing special that I could take over to my case. I actually don't understand how retraction is changed layerwise when executing the built-in retraction calibration, there is no entry in the "layer change gcode" section of the printer configuration causing the retraction changes. Anyway.

I am afraid that I would find more irregularities causing stringing and thought, perhaps try a "nightly" version. Nevertheless, in the release notes I could not find any fixes to these issues, but perhaps I missed something. I cannot believe that such bugs that spoil many prints are allowed to live long.

My question now is, are these elementary issues still open indeed??

**Comment by 4eIIIuP** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/42bed803-f1e8-4483-8660-2fb04d1ef70a)
Version 2.0 RC. Rollbacks work fine with z hop. As you can see in the photo, the slicing was performed in Auto mode and you can see the zhope both spiral and conventional.

On account of kickbacks. Have you calibrated the temperature? Was the rollback calibration performed on adjacent cylinders or were the models spaced to the maximum distance?
